### PR TITLE
[Ready for review] Few-Shot Learning: add EFL strategy

### PR DESCRIPTION
**Pull request by tianxin1860**
### PR types

New features
### PR changes

Models
### Description

1. Add EFL strategy of Few-Shot Learning
2. Support 9 Tasks of FewCLUE based on EFL strategy

Todo:
1. README.md
2. Evaluate EFL performance on 9 FewCLUE tasks

### CODE REVIEW

File: examples/few_shot/efl/data.py [hunk 1]
```diff
@@ -0,0 +1,595 @@
...
+                for idx, cantidate in enumerate(candidates):
+                    new_example = dict()
+                    new_example["sentence1"] = example["content"]
+                    new_example["sentence2"] = "位置#idiom#处的成语应该填写" + cantidate
+                    examples.append(new_example)
+
+        return MapDataset(examples)
+
+
+Processor_dict = {
```
Comment (ZeyuChen) [inline examples/few_shot/efl/data.py]: 这个变量命名形式不太符合规范
要么ProcessDict 作为类，要么processor_dict作为对象

File: examples/few_shot/efl/data.py [hunk 2]
```diff
@@ -0,0 +1,595 @@
...
+        label = example["label"]
+
+    if is_test:
+        return src_ids, token_type_ids
+    else:
+        return src_ids, token_type_ids, label
+
+
+class DataProcessor(object):
+    """Base class for data converters for sequence classification data sets."""
```
Comment (ZeyuChen) [inline examples/few_shot/efl/data.py]: data sets -> datasets

File: examples/few_shot/efl/predict.py [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+
+import numpy as np
+import paddle
+import paddle.nn.functional as F
+
+import paddlenlp as ppnlp
+from paddlenlp.data import Stack, Tuple, Pad
+from paddlenlp.datasets import load_dataset
+
+from data import create_dataloader, convert_example, Processor_dict
```
Comment (ZeyuChen) [inline examples/few_shot/efl/predict.py]: processor_dict准确点，应该是个变量

**Review by ZeyuChen — COMMENTED** [review]
Fix some typo problem

File: examples/few_shot/efl/data.py (hunk 1 above)
Comment (tianxin1860) [reply examples/few_shot/efl/data.py]: 已修正，thanks.

File: examples/few_shot/efl/data.py (hunk 2 above)
Comment (tianxin1860) [reply examples/few_shot/efl/data.py]: 已修正，thanks

File: examples/few_shot/efl/predict.py (hunk 3 above)
Comment (tianxin1860) [reply examples/few_shot/efl/predict.py]: 已修正，thanks.

**Review by ZeyuChen — APPROVED** [review]
LGTM

**Pull request merged**